tests/src/Common/test_dependencies/project.json

```diff
@@ -1,9 +1,9 @@
 {
   "dependencies": {
-    "Microsoft.NETCore.Platforms": "2.0.0-preview1-25214-03",
-    "Microsoft.Private.CoreFx.NETCoreApp": "4.4.0-preview1-25214-03",
-    "System.Memory": "4.4.0-preview1-25214-03",
-    "System.Runtime.CompilerServices.Unsafe": "4.4.0-preview1-25214-03"
+    "Microsoft.NETCore.Platforms": "2.0.0-preview1-25220-02",
+    "Microsoft.Private.CoreFx.NETCoreApp": "4.4.0-preview1-25220-02",
+    "System.Memory": "4.4.0-preview1-25220-02",
+    "System.Runtime.CompilerServices.Unsafe": "4.4.0-preview1-25220-02"
```

Comment:
@dagood this PR seems to have deleted System.Security.Permissions from this file - https://github.com/dotnet/coreclr/blob/master/tests/src/Common/test_dependencies/project.json#L7. Do you know why that happened?